extractors/reddit/reddit.go

```diff
@@ -11,7 +11,6 @@ import (
 )
 
 var (
-	PostId       string
 	redditMP4API = "https://v.redd.it/"
 	audioURLPart = "/DASH_audio.mp4"
```

Comment:
I read the discussion here https://github.com/iawia002/lux/issues/38 and i didn't figure out how to implemented it yet